src/Java.md

```diff
@@ -173,7 +173,7 @@ It would be appreciated if there are any Java VTK experts who could convert any
 | -------------- | ---------------------- | ------------- | ------- |
 [FullScreen](/Java/Utilities/FullScreen) | vtkRenderWindow | Example to Display Actors in a Full Screen Render Window
 [TimerLog](/Java/Utilities/TimerLog) | vtkTimerLog | Example to Demonstrate Timer support and logging.
-[UnknownLengthArray](/Java/Utilities/UnknownLengthArray) | vtkFloatArray | |
+[UnknownLengthArray](/Java/Utilities/UnknownLengthArray) | vtkFloatArray | Dynamic, self-adjusting array of float
```

Comment:
It does not go here, it goes in the Array section.

Reply:
I have updated.  Please check.